js/samples/04.ai-apps/h.datasource-azureOpenAI/README.md

```diff
@@ -139,6 +107,7 @@ As this sample uses managed identity, you must enable assign Cognitive Service O
 
 Now, your published custom copilot Teams app is ready for use.
 
+## Supplementary Details and Tips
```

Comment:
In other samples I think we use "Other resources" but this change is up to you

Reply:
I'll keep this as is if it's acceptable. Thank you for the info